internal/worker_server/server.go

```diff
@@ -0,0 +1,58 @@
+package worker_server
```

Comment:
minor: package names should not have underscores

```suggestion
package workerserver
```
ref. https://google.github.io/styleguide/go/decisions.html#package-names

Reply:
OK